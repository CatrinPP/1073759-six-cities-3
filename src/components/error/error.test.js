import React from 'react';
import renderer from 'react-test-renderer';
import Error from './error.jsx';


it(`Should render Error correctly`, () => {
  const tree = renderer
    .create(<Error />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
