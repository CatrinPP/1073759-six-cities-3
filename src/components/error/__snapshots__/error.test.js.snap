// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Error correctly 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      >
        <a
          className="header__logo-link header__logo-link--active"
        >
          <img
            alt="6 cities logo"
            className="header__logo"
            height="41"
            src="img/logo.svg"
            width="81"
          />
        </a>
      </div>
    </div>
    <h1>
      Error
    </h1>
    <p>
      Server error has occurred
    </p>
  </div>
</header>
`;
